website/docs/examples/deno.md

```diff
@@ -1,36 +1,37 @@
 ---
-sidebar_position: 2
+sidebar_position: 3
 sidebar_label: Using with Deno
 ---
 
 # Using with Deno
 
-To be able to use Discordeno with Deno there is one workaround needed if you are using the gateway package.
-We need to make Deno use the websocket from Deno instead of the one which is used in node, because the npm support in Deno cannot use our websocket correctly.
+Discordeno supports Deno by using the npm: specifier for your import.
 
-To do it you can use a workaround provided here: https://nest.land/package/katsura/files/src/discordenoFixes/gatewaySocket.ts
```

Comment:
Is this fix no longer required?

Reply:
i tried with latest and it was connecting fine without it. If it has long term i have no clue

Reply:
sounds good to me.